hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -934,6 +946,18 @@ public static boolean scmBootstrap(OzoneConfiguration conf)
                 + "is {}", persistedClusterId, fetchedId);
         return false;
       }
+
+      // Initialize security if security is enabled later.
+      if (OzoneSecurityUtil.isSecurityEnabled(conf)
+          && scmStorageConfig.getScmCertSerialId() == null) {
+        HASecurityUtils.initializeSecurity(scmStorageConfig, conf,
```

Comment:
the two code sections are same, can we move them to a common function ?

Reply:
Its better to move this a a static utility function.

Reply:
Done